### PR TITLE
RDM chapter update

### LINKED ISSUE #740: list of DMP tools
### Summary

https://the-turing-way.netlify.com/rdm/rdm.html mentions DMPonline (which is awesome) but my understanding from today's discussion at https://gitter.im/alan-turing-institute/the-turing-way?at=5dbc32b314d55a3785b5a20f that there are multiple DMP tools available:

- https://dmponline.dcc.ac.uk
- https://dmptool.org
- https://argos.openaire.eu/home
- https://ds-wizard.org
- https://github.com/oblassers/dmap

### What needs to be done?

- [ ] Weave the list above into the existing text at https://the-turing-way.netlify.com/rdm/rdm.html

### Who can help?

Someone who knows something about DMP tools.

---

### Updates

None yet. Please watch this space. 😄

### LINKED ISSUE #895: Fix overlap between two similar sections in different chapters ("barriers to data sharing")
### Summary

There are two sections titled "Barriers to data sharing" which could be combined into one as they discuss very similar points
* Chapter 3. Open Research (https://the-turing-way.netlify.com/open_research/01/opendata.html#barriers-to-data-sharing)
* Chapter 7. Research Data Management (https://the-turing-way.netlify.com/rdm/rdm.html#barriers-to-data-sharing)

### What needs to be done?

- [ ] Decide which chapter of the book this part best belongs to (they're currently in chapters 3 and 7)
- [ ] Combine the two sections into one
- [ ] Proofreading

### Who can help?

- [ ] Anyone

---

### Updates

None yet

### LINKED ISSUE #1122: Add checklist to the research compendia chapter
### Summary

* @annakrystalli in [her tweet](https://twitter.com/annakrystalli/status/1263377126625075200?s=20) just pointed to this resource that has a comprehensive checklist for the [research compendia chapter](https://the-turing-way.netlify.app/research_compendia/research_compendia.html) and its review: https://2020.emnlp.org/blog/2020-05-20-reproducibility

### What needs to be done?

- [ ] Edit and improve the current checklist using these resources
- [ ] Cite these resources and authors in the chapter

### Who can help?

* Anyone who has some time and wants to learn about research compendia
* The authors of this post are welcome to become the contributors as well (check with @annakrystalli)

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### LINKED ISSUE #1293: Review chapter: Research Data Management landing page
### Summary

Following a general review of the [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html) chapter in the [Guide for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html), there seems to be an opportunity for greater clarity and increased usability for researchers in the field. Below I outline some ideas that can be implemented in the chapter's [landing page](https://the-turing-way.netlify.app/reproducible-research/rdm.html). Please feel free to refute any of the suggestions, and tackle only one of the tasks (rather than all) if you want!

### What needs to be done?

- [ ] *For the [benefits section](https://the-turing-way.netlify.app/reproducible-research/rdm.html#what-are-the-benefits-of-research-data-management), could there be an introductory paragraph? Something along the lines of "Whilst it can not be denied that research data management is a complex exercise and comes at a financial cost (consider UK Data Service's commentary and tool for [data management costing](https://www.ukdataservice.ac.uk/manage-data/plan/costing)), there are clear benefits to integrating RDM into your research project. Some benefits are listed below:".*
- [x] *The five [benefits of data management](https://the-turing-way.netlify.app/reproducible-research/rdm.html#what-are-the-benefits-of-research-data-management) can be expanded on/justified and linked to further resources (or other sections within the book, like the third bullet point does).*
- [x] *The ["What is Data" section](https://the-turing-way.netlify.app/reproducible-research/rdm.html#what-is-data) offers useful questions for researchers to identify data, but the distinction between data science and other fields of inquiry can be toned down.*
- [ ] *Add an image or a clearer description of the research data lifecycle in the [last section](https://the-turing-way.netlify.app/reproducible-research/rdm.html#the-research-data-lifecycle-a-model-for-data-management)*

### Who can help?

* *Anybody with the time to reflect on data management!*

### LINKED ISSUE #1296: Review chapter: Data storage and organisation
### Summary

Following a general review of the [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html) chapter in the [Guide for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html), there seems to be an opportunity for greater clarity and increased usability for researchers in the field. Below I outline some ideas that can be implemented in the [Data storage and organisation subchapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html). Please feel free to refute any of the suggestions, and tackle only one of the tasks (rather than all) if you want!

### What needs to be done?

- [x] *Rename the subchapter "Storage and Backup" to reflect the title of the subchapter provided in Netlify's site map (left-most column).*
- [ ] *[Where to store data](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#where-to-store-data) could define (or link to a definition of) "network drives".*
- [ ] *Mention the Turing's [Data Safe Haven](https://www.turing.ac.uk/research/research-projects/data-safe-havens-cloud) as a framework for a secure IT infrastructure in [Where to store data](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#where-to-store-data).*
- [ ] *Merge [Data organisation examples](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#data-organisation-examples) into [Data organisation](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#data-organisation) by linking the tools into the two bullet points.*
- [ ] *Add a mention of spreadsheets as a tool for data organisation, linking to [the next chapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-spreadsheets.html).*
- [ ] *Finish [Data organisation](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#data-organisation) section with a sentence explaining that metadata (linking to the [pertinent subchapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html)) can also help organise files, as well as keeping consistent naming conventions (which links nicely with next section!).*
- [ ] *Correctly format bullet points in [File naming conventions](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#file-naming-conventions).*
- [ ] *Explain [best practice for data backup](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-storage.html#backups), which is to follow institution's guidelines, as well as considerations of backup automation (see [UK Data Service's tools](https://www.ukdataservice.ac.uk/manage-data/store/backup.aspx) for this, for example), amongst others.*

### Who can help?

* *Anybody with the time to reflect on data management!*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### LINKED ISSUE #1297: Review chapter: Sharing and archiving data
### Summary

Following a general review of the [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html) chapter in the [Guide for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html), there seems to be an opportunity for greater clarity and increased usability for researchers in the field. Below I outline some ideas that can be implemented in the [Sharing and archiving data subchapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html). Please feel free to refute any of the suggestions, and tackle only one of the tasks (rather than all) if you want!

### What needs to be done?

- [ ] *Integrate the [Documentation and metadata subchapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html) into this one (as it is relatively short and fits well immediately after this subchapter's [first section](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#motivations-for-sharing-data)).*
- [ ] *Link [step 4](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#step-4-choose-a-licence-and-link-to-your-paper-and-code) to book on [Licensing](https://the-turing-way.netlify.app/reproducible-research/licensing.html).*
- [ ] *In the [Consent section](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#consent), a link to [UK Data Service's page](https://www.ukdataservice.ac.uk/manage-data/legal-ethical/consent-data-sharing/consent-forms) on the matter has downloadable templates.*
- [ ] *It may be useful to define [Consent](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#consent) following UKRI's [discussion](https://esrc.ukri.org/funding/guidance-for-applicants/research-ethics/frequently-raised-questions/what-is-freely-given-informed-consent/) around "freely given informed consent".*
- [ ] *Split the [National and commercially sensitive data section](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-sharing.html#national-and-commercially-sensitive-data) into two paragraphs, as they are very different arguments (and each worthy of lengthy discussion).*

### Who can help?

* *Anybody with the time to reflect on data management!*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### LINKED ISSUE #1298: Review chapter: RDM Checklist
### Summary

Following a general review of the [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html) chapter in the [Guide for Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/reproducible-research.html), there seems to be an opportunity for greater clarity and increased usability for researchers in the field. Below I outline some ideas that can be implemented in the [Checklist subchapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-resources.html). Please feel free to refute any of the suggestions, and tackle only one of the tasks (rather than all) if you want!

### What needs to be done?

- [ ] *Shorten [checklist items](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-resources.html#checklist) for focus (link to sections in book for people to find more info).*
- [ ] *Remove "Have a plan!" and add items for each of this line's suggestions.*
- [ ] *Clarify the lne on "create the data you want to see in the world" to something along the lines of "Create clear reusable data alongside helpful metadata".*
- [ ] *Share link to [DMPOnline's checklist](https://dmponline.dcc.ac.uk/files/DMP_Checklist_2013.pdf).*

### Who can help?

* *Anybody with the time to reflect on data management!*

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### LINKED ISSUE #1970: [BUG] Duplicate content in Open Data section
### Summary

Hi all!
I noticed a duplicate in the paragraphs [Privacy and Data Protection](https://the-turing-way.netlify.app/reproducible-research/open/open-data.html#privacy-and-data-protection) and [Anonymization](https://the-turing-way.netlify.app/reproducible-research/open/open-data.html#anonymisation) of the [Open Data](https://the-turing-way.netlify.app/reproducible-research/open/open-data.html) section of the book. In particular, the following piece of text appears twice:

> Individuals must be protected from (re)identification through their data Anonymisation of the data may be sufficient in some cases, but ensuring that re-identification is not possible is becoming increasingly difficult. It might even be impossible due to technical progress, growing computational power and – ironically – more open data.
> For example, re-identification may be possible via data-mining of accessible data and so-called quasi-identifiers, a set of (common) properties that are – in their combination – so specific that they can be used to identify an individual. Preserving privacy may still be possible if partial or generalised datasets are provided, like age groups instead of birth date, or only the first two digits of postal codes. It may also be possible to provide the data in such a format that researchers can query whilst keeping the data itself closed. For example, a user may be able to send a query to retrieve the mean of a dataset, but not be able to access any of the individual data points. 

I don't know if this is intended, but I think it would be clearer if the duplicate content was only included in the Anonymization paragraph (which seems to me more coherent with the content) and the Privacy and Data Protection paragraph included only a part of the duplicate content (if deemed necessary) or entirely different content.

### What needs to be done?

- [ ] changing the content of Privacy and Data Protection paragraph in the Open Data section

### Who can help?

* editors
* me?

---

### LINKED ISSUE #2037: [Bug] Add Fair Cookbook as reference in the FAIR (RDM) chapter
### Summary

* The FAIR cookbook should be added as a reference in the [FAIR chapter](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-fair.html), as well as linked to the [impact story by Susanna](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-stories.html)

### What needs to be done?

- [ ] As a collaborating project, we want to highlight the Fair cookbook resources and also find ways for people to get involved there.

### Who can help?

* Anyone interested to add this
* Understanding of FAIR principle is not necessary but useful

---

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Fixes #1122
Fixes #1298
Fixes #740
Fixes #2037
Fixes #1970
Fixes #895
Fixes #1297
Fixes #1296
Fixes #1293

### List of changes proposed in this PR (pull-request)

* Add the [10 Things for Curating Reproducible and FAIR Research](https://curating4reproducibility.org/10things/) to the data curation section.
* Add documentation resources
* Fix a typo in the data citation section
* Add resources on general Research Data Management

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by EstherPlomp** [comment]
@all-contributors please add @pdurbin for content

**Comment by EstherPlomp** [comment]
@all-contributors please add @sjDCC for content

**Comment by EstherPlomp** [comment]
@all-contributors please add @flavio2018 for bug

**Comment by EstherPlomp** [comment]
https://github.com/all-contributors please add @jspickering for bug

**Comment by EstherPlomp** [comment]
@all-contributors please add @jspickering for bug

File: book/website/reproducible-research/rdm/rdm-sharing.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ A few public data repositories are [Zenodo](https://zenodo.org/), [Figshare](htt
 So that others know what they can do with your data, you need to apply a licence [{term}`def<License>`] to your data.
 The most commonly used licences are [Creative Commons](https://creativecommons.org/choose/), [Open Government Licence](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/), or an [Open Data Commons Attribution License](https://opendatacommons.org/licenses/by/index.html).
 To get maximum value from data sharing, make sure that your paper and code both link to your data, and vice versa, to allow others understand your project better.
+See {ref}`rr-licensing` for more information.
```
Comment (pdurbin) [inline book/website/reproducible-research/rdm/rdm-sharing.md:46]: I can't comment on line 39 above but you are welcome to list [Harvard Dataverse](https://dataverse.harvard.edu/) along with Zenodo, Figshare, and Dryad. We're all participating in the NIH's Generalist Repository Ecosystem Initiative (GREI) together: https://datascience.nih.gov/data-ecosystem/generalist-repository-ecosystem-initiative

Here is the full list from the NIH: https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/generalist-repositories (via https://sharing.nih.gov/data-management-and-sharing-policy/sharing-scientific-data/selecting-a-data-repository ).

**Review by pdurbin — COMMENTED** [review]
Just a suggestion to add Harvard Dataverse to the list of public data repositories.

File: book/website/reproducible-research/rdm/rdm-sharing.md (hunk 1 above)
Comment (EstherPlomp) [reply book/website/reproducible-research/rdm/rdm-sharing.md:47]: Added - thanks!

File: book/website/reproducible-research/rdm/rdm-checklist.md [hunk 2]
```diff
@@ -5,27 +5,23 @@ Below follow two checklists, one for individual researchers and one for research
...
-Keep the documentation with the data!
-
-- Create the data you want to see in the world. 
-Imagine someone was about to give you a dataset that you needed to analyse well in order to get that job you have been dreaming about. 
-What would you want it to look like? That is how yours should look.
+- Document everything and keep the documentation with the data (see {ref}`Documentation and Metadata<rr-rdm-metadata>`) to ensure your data is reusable. 
 
 - Try not to re-invent the wheel. 
-Before you start creating a new schema, storage format or naming protocol, have a quick google or ask your colleagues. 
+Before you start creating a new schema, storage format or naming protocol, have a quick google or ask your colleagues (see {ref}`Documentation and Metadata<rr-rdm-metadata>`).
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm/rdm-checklist.md:16]:
```suggestion
Before you start creating a new schema, storage format or naming protocol, try to find an existing tool: have a (quick) internet search or ask your colleagues (see {ref}`Documentation and Metadata<rr-rdm-metadata>`).  

```

File: book/website/reproducible-research/rdm/rdm-checklist.md [hunk 3]
```diff
@@ -5,27 +5,23 @@ Below follow two checklists, one for individual researchers and one for research
...
-Imagine someone was about to give you a dataset that you needed to analyse well in order to get that job you have been dreaming about. 
-What would you want it to look like? That is how yours should look.
+- Document everything and keep the documentation with the data (see {ref}`Documentation and Metadata<rr-rdm-metadata>`) to ensure your data is reusable. 
 
 - Try not to re-invent the wheel. 
-Before you start creating a new schema, storage format or naming protocol, have a quick google or ask your colleagues. 
+Before you start creating a new schema, storage format or naming protocol, have a quick google or ask your colleagues (see {ref}`Documentation and Metadata<rr-rdm-metadata>`).  
 Something that is already being used is likely to be better in the long run.
 
+- Follow any required procedures by your research group, institute, country (ethics, integrity codes, policies on data management/open science)
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm/rdm-checklist.md:19]:
```suggestion
- Follow any procedures required by your research group, institute, or country (ethics, integrity codes, policies on data management/open science)
```

File: book/website/reproducible-research/rdm/rdm-dmp.md [hunk 4]
```diff
@@ -78,6 +78,17 @@ If you do not select a license others will not be able to reuse your data withou
...
 You can use this [checklist](https://ukdataservice.ac.uk/learning-hub/research-data-management/plan-to-share/checklist/) to see if you have everything covered in your Data Management Plan. 
 
+(rr-rdm-dmp-tools)=
+
+## DMP tools 
+There are several platforms or tools that you can use to set up your Data Management Plan: 
+
+* [DMPonline](https://dmponline.dcc.ac.uk)
+* [DMPtool](https://dmptool.org)
+* [ARGOS](https://argos.openaire.eu/home)
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm/rdm-dmp.md:88]:
```suggestion
* [ARGOS](https://argos.openaire.eu/home)
* [DMPonline](https://dmponline.dcc.ac.uk)
* [DMPtool](https://dmptool.org)

```

Comment (jcolomb) [reply book/website/reproducible-research/rdm/rdm-dmp.md:88]: (start with the best of them :) )

File: book/website/reproducible-research/rdm/rdm-dmp.md [hunk 5]
```diff
@@ -78,6 +78,17 @@ If you do not select a license others will not be able to reuse your data withou
...
+
```
Comment (jcolomb) [inline book/website/reproducible-research/rdm/rdm-dmp.md:89]: * [RDMO](https://rdmorganiser.github.io)

**Review by jcolomb — APPROVED** [review]
small typos and minimal change

File: book/website/reproducible-research/rdm/rdm-dmp.md (hunk 5 above)
Comment (EstherPlomp) [reply book/website/reproducible-research/rdm/rdm-dmp.md:90]: Is that for creating DMPs specifically?

Comment (jcolomb) [reply book/website/reproducible-research/rdm/rdm-dmp.md:90]: yes

**Pull request merged**